go.mod

```diff
@@ -5,7 +5,7 @@ go 1.22.8
 require (
 	github.com/VictoriaMetrics/fastcache v1.12.1
 	github.com/antithesishq/antithesis-sdk-go v0.3.8
-	github.com/ava-labs/avalanchego v1.11.12-rc.2.0.20241003220930-43d5b435a644
+	github.com/ava-labs/avalanchego v1.11.13-0.20241018024240-22fb2f016633
```

Comment:
could you add a comment to the PR or in the description a link or summary of what has changed in avalanchego?